### PR TITLE
fix(signature-collection): fixes after test session 2

### PR DESCRIPTION
**Pull request by albinagu**
## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/shared/connected/src/lib/SignatureLists/useGetSignatureLists.ts [hunk 1]
```diff
@@ -25,10 +28,47 @@ export const GetCurrentCollection = gql`
     }
   }
 `
+export const GetOpenLists = gql`
+  query allOpenLists {
+    signatureCollectionAllLists {
```
Comment (alexdiljar) [inline libs/shared/connected/src/lib/SignatureLists/useGetSignatureLists.ts]:
```suggestion
    signatureCollectionAllOpenLists {
```
Use this instead, this one is meant for admins.

**Review by alexdiljar — APPROVED** [review]
nice!

File: libs/shared/connected/src/lib/SignatureLists/SignatureLists.tsx [hunk 2]
```diff
@@ -16,19 +19,48 @@ export const SignatureLists: FC<
...
         <Box marginBottom={3}>
           <Text variant="h4">
             {t('title', 'Frambjóðendur sem hægt er að mæla með')}
           </Text>
         </Box>
         <Stack space={4}>
-          {collection?.candidates?.length > 0 &&
+          {/* if collection time is over yet there are still open lists, show them */}
+          {new Date() > new Date(collection.endTime) && openLists?.length ? (
+            openLists?.map((list: any) => {
```
Comment (RunarVestmann) [inline libs/shared/connected/src/lib/SignatureLists/SignatureLists.tsx:36]: Having this typed as any is not desirable, I'd appreciate it if we could add a correct type here 🙃

**Review by RunarVestmann — APPROVED** [review]

File: libs/shared/connected/src/lib/SignatureLists/SignatureLists.tsx (hunk 2 above)
Comment (albinagu) [reply libs/shared/connected/src/lib/SignatureLists/SignatureLists.tsx:36]: 💯

**Pull request merged**